include/mrisutils.h

```diff
@@ -51,11 +51,11 @@ typedef struct {
   int nsubjects;
   MRIS *targsurfreg; // an actual surface to use as the target registration
   int  icoorder; // only use if targsurfreg not spec
-  char *targsubject; // only use if targsurfreg and icoorder not spec
-  char *hemi; // lh or rh
-  char *surfname; // eg, white
-  char *surfregname; // eg, sphere.reg
-  char *xform_name; // eg, talairach.xfm
+  const char *targsubject; // only use if targsurfreg and icoorder not spec
+  const char *hemi; // lh or rh
```

Comment:
Given that the values have to be `lh` or `rh` an `enum class` with `ToString()` and `Parse()` methods might be better.